applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -409,7 +409,7 @@ int bt_mgmt_init(void)
 			return ret;
 		}
 
-		if (IS_ENABLED(CONFIG_TESTING_BLE_ADDRESS_RANDOM)) {
+		if (IS_ENABLED(CONFIG_TESTING_BLE_BONDING_CLEAR)) {
```

Comment:
I did not quite get why we could not use CONFIG_TESTING_BLE_ADDRESS_RANDOM here anymore. Where is this used now?

Reply:
It is just to clear the bonding info only for the scan delegator test. It makes the system more stable from resets and  timing of starting the systems.